Buildings/Controls/OBC/CDL/Logical/Validation/Proof.mo

```diff
@@ -0,0 +1,152 @@
+          {-22,80}},color={255,0,255}));
+  connect(latInp2.y, pro1.u_s) annotation (Line(points={{-58,-50},{-40,-50},{-40,
+          40},{-22,40}}, color={255,0,255}));
+  connect(latInp1.y, pro2.u_s) annotation (Line(points={{-58,20},{-30,20},{-30,0},
+          {-22,0}}, color={255,0,255}));
+  connect(latInp2.y, pro4.u_s) annotation (Line(points={{-58,-50},{-40,-50},{-40,
+          -80},{-22,-80}}, color={255,0,255}));
+  connect(latInp1.y, pro3.u_s) annotation (Line(points={{-58,20},{-30,20},{-30,-40},
+          {-22,-40}},color={255,0,255}));
+  connect(latInp.y, pro.u_m) annotation (Line(points={{-58,64},{-10,64},{-10,68}},
+          color={255,0,255}));
+  connect(latInp1.y, pro4.u_m) annotation (Line(points={{-58,20},{-30,20},{-30,-96},
+          {-10,-96},{-10,-92}}, color={255,0,255}));
+  connect(latInp2.y, pro3.u_m) annotation (Line(points={{-58,-50},{-40,-50},{-40,
+          -56},{-10,-56},{-10,-52}}, color={255,0,255}));
+  connect(latInp2.y, pro2.u_m) annotation (Line(points={{-58,-50},{-40,-50},{-40,
+          -16},{-10,-16},{-10,-12}}, color={255,0,255}));
+  connect(latInp1.y, pro1.u_m) annotation (Line(points={{-58,20},{-30,20},{-30,24},
+          {-10,24},{-10,28}}, color={255,0,255}));
+  connect(latInp3.y, pro5.u_m)
+    annotation (Line(points={{62,40},{70,40},{70,68}}, color={255,0,255}));
+  connect(latInp4.y, pro5.u_s)
+    annotation (Line(points={{42,80},{58,80}}, color={255,0,255}));
+annotation (
+    experiment(StopTime=10.0, Tolerance=1e-06),
+    __Dymola_Commands(
+      file="modelica://Buildings/Resources/Scripts/Dymola/Controls/OBC/CDL/Logical/Validation/Proof.mos" "Simulate and plot"),
+    Documentation(
+      info="<html>
+<p>
+Validation test for the block
+<a href=\"modelica://Buildings.Controls.OBC.CDL.Logical.Proof\">
+Buildings.Controls.OBC.CDL.Logical.Proof</a>. The following tests are implemented:
+</p>
+<ul>
+<li>
+The instance <code>pro</code> tests the case in which
+both boolean inputs change simultaneously, and hence both outputs will be
+<code>false</code>.
+</li>
+<li>
+The instances <code>pro2</code> and <code>pro3</code> use a different value
+for <code>feedbackDelay</code>. They test the case in which
+both the inputs change from <code>true</code> to <code>false</code>. However,
+after the input <code>u_m</code> changes, the input <code>u_s</code> remains
+<code>true</code> for a time that is longer than
+<code>feedbackDelay</code>. The output <code>yProTru</code> will be <code>true</code>.
+</li>
+<li>
+The instances <code>pro1</code> and <code>pro4</code> use a different value
+for <code>feedbackDelay</code>. They test the case in which
+both the inputs change from <code>true</code> to <code>false</code>. However,
+after the input <code>u_s</code> changes, the input <code>u_m</code> remains
+<code>true</code> for a time that is longer than
+</li>
+<li>
+It also tests the case when the measured input <code>u_m</code> cannot stay stable
```

Comment:
What is "It" in this description? I revised all the other ones to start with the instance names and then describe what they do, but for this item I don't know what it means. I don't think it is `pro5` as its outputs do not remain true. Either there is a bug in the `Proof` block, or the example is not good, or the description is not about `pro5`, but then there is no explanation for `pro5`.
![image](https://user-images.githubusercontent.com/1329080/231538632-fa74933f-c558-4d27-9045-d97075e3622a.png)

Reply:
Thanks for the comments. It indeed is a bug in the block. I have corrected the implementation and updated the validation model's documentation.